heartbeat/scripts/post_process_config.py

```diff
@@ -24,7 +24,13 @@
             if section_name == "Processors":
                 output += line  # include section name in output
                 output += "processors:\n"
-                output += "  - add_observer_metadata: ~\n"
+                output += "  - add_observer_metadata: \n"
+                output += "      # Optional, but recommended geo settings for the location heartbeat is running in \n"
+                output += "      # geo: \n"
```

Comment:
We normally leave out the space in front of config options to differentiate from comments and make uncommenting easier. Same below.